### ISSUE TITLE
Rich Editor - Unable to select around embed blots

### ISSUE THREAD

**Issue by charrondev** [vanilla/vanilla#6999]
It's not possible to select around embed blots if they are in the first or last position.

Shift + Arrows should also work here.

Maybe look into allow a transparent overlay of some selectable element?

**Comment by DaazKu** [comment]
I stumbled on that issue myself by doing the following:

1. Quote a comment
1. Write some text
1. CTRL + A
1. DELETE

Expected result: The whole content of the editor should be deleted.
Actual result: Nothing happen.

**Comment by DaazKu** [comment]
That's a bug.
